src/_P184_Triac.ino

```diff
@@ -0,0 +1,380 @@
+// This plugin detect zerocross and trigger a triac. Used for power electronics.
+// Comon uses are AC light dimming or AC Fan speed control. 
+
+#include "_Plugin_Helper.h"
+
+#ifdef USES_P184
+#define PLUGIN_184
+#define PLUGIN_ID_184     184           // plugin id
+#define PLUGIN_NAME_184   "Triac" // "Plugin Name" is what will be dislpayed in the selection list
```

Comment:
The name convention is to use a category, a dash (-) and the plugin name, so this could be `Output - Triac`.

(The Output category seems most fitting for this plugin)